2024-02-13-credentialscg-crossmeeting/README.md

```diff
@@ -0,0 +1,3 @@
+For notes see the [w3c credentials CG meetings repo](https://github.com/w3c-ccg/meetings). 
```

Comment:
Is there a specific meeting note that can be linked to? Perhaps https://w3c-ccg.github.io/meetings/2024-02-13/ ?

Reply:
good catch.  i think i opened this between the meeting ending and those minutes going up on that website!